content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -149,20 +149,15 @@ until disk usage reaches the `LowThresholdPercent` value.
 As a beta feature, you can specify the maximum time a local image can be unused for,
 regardless of disk usage. This is a kubelet setting that you configure for each node.
 
-To configure the setting, you need to set a value for the `imageMaximumGCAge` 
+To configure the setting, you need to set a value for the `imageMaximumGCAge`
 field in the kubelet configuration file.
 
-The value is specified as a Kubernetes _duration_; 
-Valid time units for the `imageMaximumGCAge` field in the kubelet configuration file are:
-- "ns" for nanoseconds
-- "us" or "µs" for microseconds
-- "ms" for milliseconds
-- "s" for seconds
-- "m" for minutes
-- "h" for hours
+The value is specified as a Kubernetes {{< glossary_tooltip text="duration" term_id="duration" >}}.
+See [duration](/docs/reference/glossary/?all=true#term-duration) in the glossary
+for more details.
```

Comment:
Delete line 156-157?

Reply:
I added this because I thought it would help readers. Does it not?